pkgs/by-name/co/coredns/package.nix

```diff
@@ -32,7 +32,7 @@ in buildGoModule rec {
 
   # Override the go-modules fetcher derivation to fetch plugins
   modBuildPhase = ''
-    for plugin in ${builtins.toString (attrsToPlugins externalPlugins)}; do echo $plugin >> plugin.cfg; done
+    for plugin in ${builtins.toString (attrsToPlugins externalPlugins)}; do sed -i "$plugin" plugin.cfg; done
```

Comment:
When adding multiple plugins, the plugin placed at the back may not get the expected absolute position. How to solve this?

Reply:
we could sort the plugins such that those with higher line number are inserted first:

```
plugin A: priority=6
plugin B: priority=11
plugin C: priority=10
```

insert B, then C, then A.

Reply:
Great.